servers/quarkus-tests/src/main/java/org/projectnessie/quarkus/tests/profiles/PostgresTestResourceLifecycleManager.java

```diff
@@ -31,14 +33,35 @@
 
   private Optional<String> containerNetworkId;
 
+  public String getDefaultPostgresVersion(){
+    String postgresVersion;
+    try(InputStream input = PostgresTestResourceLifecycleManager.class.getClassLoader().getResourceAsStream("gradle.properties")) {
```

Comment:
There's no need to read the file, see [docs](https://docs.gradle.org/current/userguide/build_environment.html) how system properties get passed through.